### ISSUE TITLE
Provide winget artifact in release

### ISSUE THREAD

**Issue by paultyng** [hashicorp/terraform-ls#114]
Related to #55 and #47, this is the new Windows package manager from Microsoft.

**Comment by radeksimko** [comment]
I agree we should do it, but just to set the expectations, this won't replace Scoop or Chocolatey work, more importantly WinGet only supports alternative sources as the single point of truth for now:

https://github.com/microsoft/winget-cli#sources

> The client is built around the concept of sources; a set of packages effectively. Sources provide the ability to discover and retrieve the metadata about the packages, so that the client can act on it.
> 
> The default source reflects that data available from the [Community repo](https://github.com/microsoft/winget-pkgs).
> 
> We plan to better support additional sources, and additional types of sources, in the future. For now, additional sources can be configured, but only one used at a time.

So even if we did publish our own source/repo for WinGet it's unlikely anyone would use it. They would have to use it exclusively and loose access to the community source.

The only thing we could do is publish updates to the community repo, but given that it's PR-driven process it's not something we can easily automate and probably shouldn't anyway.

TL;DR We are waiting for Microsoft at this point.

**Comment by radeksimko** [comment]
I can't find any more specific docs online, but it looks like additional 3rd-party sources are now supported for WinGet:
https://github.com/microsoft/winget-cli/issues/226#issuecomment-831597457

**Comment by denelon** [comment]
Third party sources are supported by the Windows Package Manager client. We are working on the reference implementation of the REST API, and it will be released soon[tm].

**Comment by Sup3rlativ3** [comment]
Hello,

Is this still something being worked on? I'm automating some desktop images and I've found that tf isn't available.

**Comment by denelon** [comment]
From the Windows Package Manager side, we still need to implement support for .zip:
* microsoft/winget-cli/issues/140

A community driven draft specification is in progress:
* microsoft/winget-cli/pull/1717

We have also released the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) to help publish manifests to the [Windows Package Manager Community App Repository](https://github.com/microsoft/winget-pkgs) with CI/CD scenarios. For organizations wishing to host their own source repository, a [reference implementation](https://github.com/microsoft/winget-cli-restsource) is available.
